CITATION.rst

```diff
@@ -1 +1 @@
-sunpy/CITATION.rst
\ No newline at end of file
+sunpy/CITATION.rst
```

Comment:
Why was this changed?

Reply:
the pre commit made these changes automatically, should I reverse it back?

Reply:
That is strange, it doesn't need to one master. Leave it in for now.

Reply:
okay now what should I do now?

Reply:
Nothing, wait for reviews and then once this PR has been approved we will merge it.